lib/perl/Genome/Model/Tools/Picard/MarkDuplicates.pm

```diff
@@ -90,8 +90,27 @@ sub get_max_filehandles_param {
 sub execute {
     my $self = shift;
 
+    my $dedup_cmd = $self->build_cmdline_string;
+    $self->run_java_vm(
+        cmd => $dedup_cmd,
+        input_files => [$self->input_file],
+        output_files => [$self->output_file, $self->metrics_file],
+        skip_if_output_is_present => 0,
+    );
+    return 1;
+}
+
+sub build_cmdline_string {
+    my ($self, $cmd) = @_;
+
+    my $dedup_cmd = $self->picard_path .'/MarkDuplicates.jar';
+    if ($self->version_compare($self->use_version, '1.122') >= 0) {
+        $dedup_cmd .= ' net.sf.picard.sam.markduplicates.MarkDuplicates';
```

Comment:
The `net.sf.` part was removed from Picard beginning with version 1.114.  Should it also be removed here?